src/CarrierLogo/index.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+import * as React from "react";
+import * as tokens from "orbit-design-token";
+
+const baseURL = "//images.kiwi.com"; // token or const
+
+const directorySizes = {
+  small: 16,
+  medium: 32,
+  large: 32,
+};
+
+const renderSizes = {
+  small: parseInt(tokens.heightIconSmall, 10),
+  medium: parseInt(tokens.heightIconMedium, 10),
+  large: 32, // replace it for token heightIconLarge
+};
+
+const retinaSizes = {
+  small: 32,
+  medium: 64,
+  large: 64,
+};
+
+type Props = {
+  size: "small" | "medium" | "large",
+  carriers: { code: string, name: string }[],
```

Comment:
I would prefer this, it's more readable:

```
type Carrier = { code: string, name: string };

type Props = {
  size: "small" | "medium" | "large",
  carriers: Carrier[],
}
```

Reply:
OK